cuppa-core/src/main/java/org/forgerock/cuppa/Cuppa.java

```diff
@@ -35,52 +18,42 @@ private Cuppa() {
      * Registers a described suite of tests to be run.
      *
      * @param description The description of the 'describe' block.
-     * @param function The 'describe' block.
+     * @param function    The 'describe' block.
```

Comment:
I dont think we should attempt to align the parameter javadoc as it takes more effort to maintain and we don't do it in any other classes.